channeldb/graph_cache.go

```diff
@@ -439,9 +439,29 @@ func (c *GraphCache) ForEachChannel(node route.Vertex,
 			channelCopy.InPolicy.ToNodeFeatures = features
 		}
 
-		if err := cb(channelCopy); err != nil {
+		channelsCopy = append(channelsCopy, channelCopy)
```

Comment:
nit: would the access using an index be even faster here? Maybe not as that's probably optimized by the compiler anyway.

Reply:
Yeah assignment is probably faster, fixed.